lib/cretonne/src/ir/dfg.rs

```diff
@@ -95,6 +95,14 @@ impl DataFlowGraph {
         vref
     }
 
+    /// Check if a value reference is valid.
+    pub fn value_is_valid(&self, v: Value) -> bool {
```

Comment:
Nice, this is a good idea.

I think that in the future, the `extended_values` table will have a way of representing dangling values that are no longer valid. We can catch those here too.